.github/workflows/sync-pr-from-fork.yml

```diff
@@ -42,6 +42,14 @@ jobs:
           git remote add contributor "https://github.com/$headRepositoryOwnerLogin/$headRepositoryName.git" || :
           contribHeadRefName="contributor-$headRepositoryOwnerLogin-$headRefName"
           git fetch --no-tags contributor +"$headRefName":"$contribHeadRefName"
+
+          # Rebase the branch with the latest master HEAD
+          git fetch origin master
```

Comment:
To be noted, I don't know whether this will work or not. I have yet to check these workflows by creating a similar setup on my own. I just thought of creating this PR beforehand to get inputs from the DevOps team.